src/style.css

```diff
@@ -1,3 +1,86 @@
-body {
+* {
```

Comment:
use only box-sizing: border-box; for `*` selector. if you need to reset default browser styles use tag selectors